ApollosProject/apollos-prototype
# Fix location finder not updating after location permission granted (Android)

# DESCRIPTION

### What does this PR do, or why is it needed?

Moves the request for permission into the LocationFinder component itself, so that it can update after the permission was denied/granted.

### What design trade-offs/decisions were made?

### How do I test this PR?

On android, with a new user account, click "select campus", accept location permissions, click a campus, and then on returning to the onboarding screen you should be able to continue. 

### What automated tests did you write?

n/a

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
